pkg/controller/servicebindingrequest/secret_test.go

```diff
@@ -21,7 +22,8 @@ func TestSecretNew(t *testing.T) {
 	f := mocks.NewFake(t, ns)
 
 	matchLabels := map[string]string{}
-	sbr := mocks.ServiceBindingRequestMock(ns, name, "", "", matchLabels, true)
+	applicationGVR := schema.GroupVersionResource{Group: "apps", Version: "v1", Resource: deploymentsGVR}
+	sbr := mocks.ServiceBindingRequestMock(ns, name, "", "", applicationGVR, matchLabels, true)
```

Comment:
Having the `deploymentsGVR` constant, `applicationGVR` can be removed.

```suggestion
	sbr := mocks.ServiceBindingRequestMock(ns, name, "", "", deploymentsGVR, matchLabels, true)
```

Reply:
Done eb3f22b